tee-worker/cli/lit_ts_integration_test.sh

```diff
@@ -75,7 +75,13 @@ if [ "$TEST" = "assertion_contracts.test.ts" ]; then
     ln -s /assertion-contracts/artifacts/contracts /ts-tests/integration-tests/contracts
 fi
 
-cd /ts-tests
+# VC assertions requires our regular file structure
+if [[ "$TEST" == *"vc"* ]]; then
+    cd /app/tee-worker/ts-tests
+else
+    cd /ts-tests
+fi
+
```

Comment:
Aren't they the same path?

both `<parahcain-dir>/tee-worker/ts/-tests` - If I calculate it right

Reply:
yes, but we need the entire folder structure for this work both locally and on CI: https://github.com/litentry/litentry-parachain/blob/e5f109f9ab6ae0935f86c3120c6e8a29c84b5e93/tee-worker/ts-tests/integration-tests/common/utils/assertion.ts#L149

Reply:
Then shall we always use `cd /app/tee-worker/ts-tests`? And only mount `/app` without mounting `ts-tests` alone?

Reply:
Btw another way is to query the runtime version via RPC - that should be easy too

Reply:
> Then shall we always use cd /app/tee-worker/ts-tests? And only mount /app without mounting ts-tests alone?

Yes, I'm in with this one but wanted to leave it for a follow-up pull requests as multiple files need update.

> Btw another way is to query the runtime version via RPC - that should be easy too
> 
> Actually it should be slightly more accurate: imagine you code has bumped the runtime version but your chain runtime hasn't been updated yet

Do you want me to close this PR and use the RPC better? If so, would you mind pointing out the RPC method name for me, please?

Reply:
cc @Kailai-Wang

Reply:
Should be this one (use `specVersion`):
<img width="970" alt="image" src="https://github.com/user-attachments/assets/c7d56600-6269-4287-9954-d56a618ab279">

Reply:
And yes please I suggest we close this PR and open a new one using RPC : )